examples/lifecycleDemo.html

```diff
@@ -115,9 +115,51 @@ <h1>Lifecycle Events Log:</h1>
             var lifecycleStatusMessageTitle = 'Lifecycle Demo App';
             var lifecycleStatusMessageId = 'lifecycleDemo-statusMsg';
 
+            function extractParams(paramStr) {
+                let result = {};
+
+                if (paramStr) {
+                    let params = paramStr.split('&');
+                    params.forEach(function(currParam) {
+                        if (currParam) {
+                            let paramTokens = currParam.split('=');
+                            let paramName = paramTokens[0];
+                            let paramValue = paramTokens[1];
+                            if (paramName) {
+                                paramName = decodeURIComponent(paramName);
+                                paramValue = paramValue ? decodeURIComponent(paramValue) : null;
+
+                                if (!result.hasOwnProperty(paramName)) {
+                                    result[paramName] = paramValue;
+                                } else if (Array.isArray(result[paramName])) {
+                                    result[paramName].push(paramValue);
+                                } else {
+                                    result[paramName] = [result[paramName], paramValue];
+                                }
+                            }
+                        }
+                    });
+                }
+
+                return result;
+            }
+
+            function getRequireManualToastDismissal() {
+                let result = null;
+
+                if (!result && window.location.search) {
```

Comment:
You dont need the !result check here anymore since the oauth params were removed.
You could moce the 'true' value check here and just return a boolean from this method

Reply:
I'm not sure if I totally understand this...is it something like
```if (window.location.search) {
                    return extractParams(window.location.search.substring(1)).requireManualToastDismissal;
                }
                return null;```?

Reply:
Yeah, that was 2 comments in 1.  Sorry.

The first comment was to remove the !result check as it was a hold-over from the previous code that first checked for the param in the hash after an authentication attempt.

The second comment is that you could move the parsing of the param value into this utility as well.  So, rather than returning the value of the parameter, you could return a boolean from this method indicating if the toasts need to be manually dismissed.

Reply:
I assume we will add to the url something like requireManualToastDismissal=true/false. So I change the code to:
```
let result = null;
if (window.location.search) {
   result = extractParams(window.location.search.substring(1)).requireManualToastDismissal;
   if (result === 'true') {
      return true;
   }
}
return false;
```
I'm wondering if this is correct?